.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       # Golang image - primary
       - image: golang:1.12.9-alpine3.10
 
-    working_directory: /code
+    working_directory: ~/repo
```

Comment:
why?

Reply:
There was some error during the CI build. Changing to build made the build work.